languages/python/reference/README.md

```diff
@@ -646,7 +646,7 @@ These will very rarely be encountered in the wild, the first because it's more o
 | [Exception handling][exception-handling]                                     | `intermediate`                                    |                        [ ]()                        |                      [ ]()                      |
 | [Exception hierarchy][exception-hierarchy]                                   | `intermediate`                                    |                        [ ]()                        |                      [ ]()                      |
 | [Exception message][exception-message]                                       | `intermediate`                                    |                        [ ]()                        |                      [ ]()                      |
-| [Exiting loops][]                                                            | `general`                                         |                   DavidGerva[ ]()                   |                      [ ]()                      |
+| [Exiting loops][]                                                            | `general`                                         | [#1432](https://github.com/exercism/v3/issues/1432)                   |                      [ ]()                      |
```

Comment:
Can we make this `[Exiting loops][exiting-loops]`, as is the link is broken and not rendering correctly.

Reply:
Done